app/models/group.rb

```diff
@@ -590,9 +590,9 @@ def self.refresh_automatic_group!(name)
       when :staff
         "SELECT id FROM users WHERE id > 0 AND (moderator OR admin) AND NOT staged"
       when :trust_level_1, :trust_level_2, :trust_level_3, :trust_level_4
-        "SELECT id FROM users WHERE id > 0 AND trust_level >= #{id - 10} AND NOT staged"
+        "SELECT id FROM users WHERE id >= -1 AND trust_level >= #{id - 10} AND NOT staged"
```

Comment:
Same comment about applying this change to staff, admin, moderators auto groups too.